<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpeedQuiz</class>
 <widget class="QWidget" name="SpeedQuiz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>461</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="progressButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>700</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>NEXT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>771</width>
     <height>561</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Answer_1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>640</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>답은 1번!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Answer_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>640</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>답은 2번!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Answer_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>640</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>답은 3번!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Answer_4">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>640</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>답은 4번!</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>671</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="Answer_1_text">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Answer_2_text">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Answer_3_text">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Answer_4_text">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>220</y>
     <width>31</width>
     <height>361</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="textDirection">
    <enum>QProgressBar::BottomToTop</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="GameOver">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>700</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>나갈래</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Retry">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>700</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>다시 할래!</string>
   </property>
  </widget>
  <widget class="QLabel" name="image">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>491</width>
     <height>361</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>progressBar</zorder>
  <zorder>label</zorder>
  <zorder>progressButton</zorder>
  <zorder>label_2</zorder>
  <zorder>Answer_1</zorder>
  <zorder>Answer_2</zorder>
  <zorder>Answer_3</zorder>
  <zorder>Answer_4</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>GameOver</zorder>
  <zorder>Retry</zorder>
  <zorder>image</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
